bundles/org.openhab.binding.comfoair/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1596 @@
+			<channel id="level1Time" typeId="level1_time" />
+			<channel id="level2Time" typeId="level2_time" />
+			<channel id="level3Time" typeId="level3_time" />
+			<channel id="freezeTime" typeId="freeze_time" />
+			<channel id="preheaterTime" typeId="preheater_time" />
+			<channel id="bypassTime" typeId="bypass_time" />
+			<channel id="filterHours" typeId="filter_hours" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="bypass">
+		<label>Bypass Values</label>
+		<channels>
+			<channel id="bypassFactor" typeId="bypass_factor" />
+			<channel id="bypassLevel" typeId="bypass_level" />
+			<channel id="bypassCorrection" typeId="bypass_correction" />
+			<channel id="bypassSummer" typeId="bypass_summer" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="preheater">
+		<label>Preheater Values</label>
+		<channels>
+			<channel id="preheaterValve" typeId="preheater_valve" />
+			<channel id="preheaterFrostProtect" typeId="preheater_frost_protect" />
+			<channel id="preheaterHeating" typeId="preheater_heating" />
+			<channel id="preheaterFrostTime" typeId="preheater_frost_time" />
+			<channel id="preheaterSafety" typeId="preheater_safety" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="ewt">
+		<label>EWT Values</label>
+		<channels>
+			<channel id="ewtTemperatureLow" typeId="ewt_temperature_low" />
+			<channel id="ewtTemperatureHigh" typeId="ewt_temperature_high" />
+			<channel id="ewtSpeed" typeId="ewt_speed" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="heater">
+		<label>Heater Values</label>
+		<channels>
+			<channel id="heaterPower" typeId="heater_power" />
+			<channel id="heaterPowerI" typeId="heater_power_I" />
+			<channel id="heaterTargetTemperature" typeId="heater_target_temperature" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="cookerhood">
+		<label>Cookerhood Values</label>
+		<channels>
+			<channel id="cookerhoodSpeed" typeId="cookerhood_speed" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="enthalpy">
+		<label>Enthalpy Values</label>
+		<channels>
+			<channel id="enthalpyTemperature" typeId="enthalpy_temperature" />
+			<channel id="enthalpyHumidity" typeId="enthalpy_humidity" />
+			<channel id="enthalpyLevel" typeId="enthalpy_level" />
+			<channel id="enthalpyTime" typeId="enthalpy_time" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="options">
+		<label>Option States</label>
+		<channels>
+			<channel id="isPreheater" typeId="is_preheater" />
+			<channel id="isBypass" typeId="is_bypass" />
+			<channel id="recuType" typeId="recu_type" />
+			<channel id="recuSize" typeId="recu_size" />
+			<channel id="isChimney" typeId="is_chimney" />
+			<channel id="isCookerhood" typeId="is_cookerhood" />
+			<channel id="isHeater" typeId="is_heater" />
+			<channel id="isEnthalpy" typeId="is_enthalpy" />
+			<channel id="isEWT" typeId="is_ewt" />
```

Comment:
I already thought about this too, but I couldn't figure out, how to properly implement this. I don't want the user having to set these on thing creation, since most users won't even know their correct values. On the other hand, values from the device can only be fetched after successful connection and therefore after creation of the thing. How would I implement this so that these values are set automatically after thing creation (initialization), but can also be updated manually from the thing configuration? Aren't the values then always overwritten with the device defaults after saving the new config (issueing a new initialization)?

Reply:
Maybe you could make them thing properties instead?

Reply:
That would probably be the best option, even if that means that these options cannot be modified by the user. But in my opinion a common user should not mess with these settings anyway and schould be set by the fitter of the appliance.
Another way that came to my mind would be a separate configuration option to set if the configuration should be pulled from the device or if these are user defined.
However, I will implement them as properties for now (BTW is there an option to set labels for properties?)

Reply:
done